src/Resolution.ts

```diff
@@ -186,6 +187,24 @@ export default class Resolution {
     return this.getNamingMethodOrThrow(domain).namehash(domain);
   }
 
+  /**
+   * returns a childhash for specific namingService
+   * @param parent -> hash for parent
+   * @param label -> hash for label
+   * @param method -> namingservice name, if omited CNS is used as default
+   */
+  childhash(parent: nodeHash, label: string, method?: string):nodeHash {
+    if (!method) method = 'CNS';
```

Comment:
I don't see any benefit of having `CNS` as a default. That may lead to incorrect assumption by someone that sees `resolution.childhash('0x83822', 'hello')  that `childhash` is the same for all services. Lets make `method` required and create an `enum NamingServiceName` that will be its type.